File: python/feldera-api-client/feldera_api_client/models/kafka_log_level.py
from enum import Enum


class KafkaLogLevel(str, Enum):
    ALERT = "alert"
    CRITICAL = "critical"
    DEBUG = "debug"
    EMERG = "emerg"
    ERROR = "error"
    INFO = "info"
    NOTICE = "notice"
    WARNING = "warning"

    def __str__(self) -> str:
        return str(self.value)

File: sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/frontend/connectors/config/KafkaLogLevel.java
package org.dbsp.sqlCompiler.compiler.frontend.connectors.config;
import com.fasterxml.jackson.annotation.JsonProperty;

public enum KafkaLogLevel {
    @JsonProperty("emerg")    Emerg,
    @JsonProperty("alert")    Alert,
    @JsonProperty("critical") Critical,
    @JsonProperty("error")    Error,
    @JsonProperty("warning")  Warning,
    @JsonProperty("notice")   Notice,
    @JsonProperty("info")     Info,
    @JsonProperty("debug")    Debug,
}
